clamav-1.4.yaml

```diff
@@ -236,6 +248,111 @@ subpackages:
             clamav-milter --version
             clamav-milter --help
 
+  - name: ${{package.name}}-docker
+    description: Scripts needed to run ClamAV in a Docker container
+    dependencies:
+      runtime:
+        - bash
+        - json-c
+        - libbz2-1
+        - libcurl-openssl4
+        - libmilter
+        - libstdc++
+        - libxml2
+        - ncurses
+        - netcat-openbsd
+        - pcre2
+        - tini
+        - tzdata
+        - zlib
+      provides:
+        - clamav-docker=${{package.full-version}}
+    pipeline:
+      - uses: git-checkout
+        with:
+          repository: https://github.com/Cisco-Talos/clamav-docker
+          expected-commit: ${{vars.CLAMAV_DOCKER_COMMIT}}
+          tag: ""
+          branch: main
+      - name: Sanity check to ensure there is corresponding version
+        runs: |
+          if [ ! -d clamav/${{vars.major-minor-version}} ]; then
+            echo "No directory for version ${{vars.major-minor-version}}"
+            exit 1
+          fi
+      - name: Sanity check to ensure folder hash is matching
+        runs: |
+          expected_hash="${{vars.CLAMAV_DOCKER_HASH}}"
+          actual_hash=$(tar -cf - clamav/${{vars.major-minor-version}} | sha256sum | cut -d' ' -f1)
+          if [ "$expected_hash" != "$actual_hash" ]; then
+            echo "Hash mismatch for version ${{vars.major-minor-version}}. Expected $expected_hash, got $actual_hash. Please run 'git clone --depth=1 git@github.com:Cisco-Talos/clamav-docker.git && cd clamav-docker && tar -cf - clamav/1.4 | sha256sum' and update the CLAMAV_DOCKER_HASH variable"
```

Comment:
This doesn't exit with a non-zero status code, so effectively it will never catch an issue

Reply:
Nice catch, done!